### PR TITLE
Add Dormant state

### PR DESCRIPTION
**Pull request by tkuhrt**
Signed-off-by: Tracy Kuhrt <tracy.a.kuhrt@accenture.com>

### CODE REVIEW

**Review by troyronda — APPROVED** [review]

**Comment by ryjones** [comment]
Maybe a new page with the state transition diagram? They’re easy to do in “dot”, a graph description language. Or as an svg

**Comment by tkuhrt** [comment]
@ryjones : I am no "dot" expert, but I have added the file and generated an SVG that I linked into the markdown file.

File: project-lifecycle.gv [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  EndOfLife [label="End of Life"]
+  //rank=same; Proposal; Incubation; Graduated; Dormant; Deprecated; EndOfLife;
+  Proposal -> Incubation
+  Incubation [color=purple];
+  Incubation -> Graduated [color = purple];
+  Graduated [color=blue];
+  Graduated -> Dormant [color = blue];
+  Dormant [color=red];
+  Dormant -> Deprecated [color = red];
+  Deprecated -> EndOfLife;
```
Comment (arsulegai) [inline project-lifecycle.gv:12]: Thinking out loud here, can we say a project that is deprecated can be revived with a new proposal. I guess that is in line with what TSC thinks.

**Review by mastersingh24 — APPROVED** [review]

**Review by lehors — COMMENTED** [review]
LGTM. Thanks!

**Pull request merged**